src/test/java/com/es/phoneshop/model/cart/HttpSessionCartServiceTest.java

```diff
@@ -0,0 +1,104 @@
+package com.es.phoneshop.model.cart;
```

Comment:
Does this test work?

I see the following test methods failing:

testAddWithStockLessQuantity
testAddWithoutSameCartItem
testAddWithSameCartItem
testAddWithSameCartItemAndStockLessQuantity